### PR TITLE
ACP2E-2266: update public docs to show how to add / change itemprop f…

### PR DESCRIPTION
**Pull request by alzota**
…or product attributes

## Purpose of this pull request

This pull request (PR) shows how to add / change itemprop on product page. This is already existing functionality and is needed for Google Search Console.

## Affected pages
https://developer.adobe.com/commerce/frontend-core/guide/layouts/xml-manage/

- ...

## Links to Magento Open Source code
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Catalog/view/frontend/layout/catalog_product_view.xml

- ...

### CODE REVIEW

**Comment by jeff-matthews** [comment]
Thanks for the contribution @alzota!

@bdenham, please review.

**Review by bdenham — CHANGES_REQUESTED** [review]
See suggested changes and let me know what you think.

File: src/pages/guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -156,6 +156,63 @@ Use the following examples to include in your own layout themes.
 </page>
 ```
 
+## Add or change itemprop global attribute for products
+
+To add `itemprop` attribute to another product attribute displayed in the `<body>` element of your layout on product page, create a theme-extending file similar to: `app/design/frontend/<Vendor>/<theme>/Magento_Catalog/layout/catalog_product_view.xml`.
+
+By default, `itemprop=description` is added for `short_description` attribute of a product.
```
Comment (bdenham) [inline src/pages/guide/layouts/xml-manage.md:163]:
```suggestion
To add the `itemprop` attribute to another product attribute displayed in the `<body>` of your product page layout:

1. Create a new theme-extension file similar to the `catalog_product_view.xml` found here: `app/design/frontend/<Vendor>/<theme>/Magento_Catalog/layout/catalog_product_view.xml`.
2. Within the new file, remove the `add_attribute` argument from the `short_description` attribute block and add it to the `description` attribute block, as shown here:

```xml
<!-- description attribute block -->
<block class="Magento\Catalog\Block\Product\View\Description" name="product.info.description" as="description" template="Magento_Catalog::product/view/attribute.phtml" group="detailed_info">
      <arguments>
          <argument name="at_call" xsi:type="string">getDescription</argument>
          <argument name="at_code" xsi:type="string">description</argument>
          <argument name="css_class" xsi:type="string">description</argument>
          <argument name="at_label" xsi:type="string">none</argument>
          <argument name="title" translate="true" xsi:type="string">Details</argument>
          <argument name="sort_order" xsi:type="string">10</argument>
          <argument name="add_attribute" xsi:type="string">itemprop="description"</argument>
      </arguments>
</block>

<!-- Example usage -->
<div class="product attribute description">
  <div class="value" itemprop="description">
    <p>Luma LifeLong Fitness Series is a world recognized, evidence based exercise program designed specifically for individuals focused on staying active their whole lives. If followed regularly, participants will see improved heart rate and blood pressure, increased mobility, reduced joint pain and overall improvement in functional fitness and health.</p>
    <ul>
      <li>10 minute warm up.</li>
      <li>30 minutes of mild aerobics.</li>
      <li>20 minutes of strength, stretch and balance.</li>
      <li>Extensive modifications for varying fitness levels.</li>
    </ul>
  </div>
</div>
```
```

Comment (bdenham) [reply src/pages/guide/layouts/xml-manage.md:163]: @alzota — Thanks for adding this new section! 

The changes I suggested above would replace the remainder of your additions. Every thing that you wrote below this comment is included — but reordered — above, to help make things a bit clearer.

Let me know what you think.

**Review by bdenham — CHANGES_REQUESTED** [review]

File: src/pages/guide/layouts/xml-manage.md (hunk 1 above)
Comment (jeff-matthews) [reply src/pages/guide/layouts/xml-manage.md:163]: @alzota, yes. You can commit @bdenham's suggestions and then we'll get this merged. Thanks for the contribution!

Comment (bdenham) [reply src/pages/guide/layouts/xml-manage.md:163]: > @bdenham looks great! thanks! can I commit your suggestion as is?

Yes, absolutely!

**Comment by jeff-matthews** [comment]
Hi @alzota,

If you accept Bruce's suggestions, we can get this merged.

**Pull request merged**